research-software-directory/RSD-as-a-service
# 544: filter software on keyword/tag using a link

# Pull request Title

Closes #544

Changes proposed in this pull request:
* The link is placed on software tag to filter software list on the tag value      

How to test:

* `make start` to rebuild all
* navigate to software page that has tag values, otherwise create new software and add keywords
* confirm that keyword is a link to software overview page and uses proper keyword filter
* perform same check on project page

## Filter software on tag from software page
![image](https://user-images.githubusercontent.com/9204081/198138396-6edd80c8-c21b-486b-921e-8affa01d846b.png)

## Filter projects on keyword from project page
![image](https://user-images.githubusercontent.com/9204081/198249529-1cf86c61-00b3-4ed6-b78b-7bb66ae1ab37.png)

PR Checklist:
